interfaces/builtin/mir.go

```diff
@@ -88,6 +88,9 @@ const mirConnectedPlugAppArmor = `
 unix (receive, send) type=seqpacket addr=none peer=(label=###SLOT_SECURITY_TAGS###),
 /run/mir_socket rw,
 /run/user/[0-9]*/mir_socket rw,
+
+# Mir uses /dev/shm for sharing memory with clients
+/{dev,run}/shm/\#* mrw,
```

Comment:
Hmm, this is pretty broad. Could mir identify that those are, at least, mir related?

Reply:
Mir is using O_TMPFILE in /dev/shm, those files *appear* to be named "#n" where n is a monotonically increasing number.

Aside from patching Mir, is there some apparmor magic that might help?

Reply:
I think this is a decision for @jdstrand - technically it's just a bit unfortunate that any client of mir interface will be able to open those files. Since they are not linked and presumably get passed through some socket I think this is okay but I'm not a security expert.

Reply:
Thanks for looking. I'm open to suggestion too, I accept this isn't ideal